QUALITY_DECLARATION.md

```diff
@@ -89,7 +89,7 @@ Most recent test results can be found [here](http://build.ros2.org/view/Epr/job/
 
 ### Feature Testing [4.i]
 
-There are currently no public features undergoing tests.
+`rmw_implementation` features enable ROS middleware configuration. Integration and system tests up the stack provide coverage.
 
 ### Public API Testing [4.ii]
```

Comment:
How about updating this one as well? There's no public API, therefore we don't need API tests.

Reply:
Sure, see https://github.com/ros2/rmw_implementation/pull/103/commits/fb834663b844ac85badeac23b11a4917ace426e3.